docs/docsite/rst/playbook_guide/playbooks_lookups.rst

```diff
@@ -21,6 +21,33 @@ You can populate variables using lookups. Ansible evaluates the value each time
       - debug:
           msg: "motd value is {{ motd_value }}"
 
+The first argument to the ``lookup`` function is required and specifies the name of the lookup plugin. If the lookup plugin is a collection the fully qualified name must be provided, since the :ref:`collections keyword<collections_keyword>` does not apply to lookup plugins.
```

Comment:
```suggestion
The first argument to the ``lookup`` function is required and specifies the name of the lookup plugin. If the lookup plugin is in a collection, the fully qualified name must be provided, since the :ref:`collections keyword<collections_keyword>` does not apply to lookup plugins.
```